R/resample.R

```diff
@@ -5,7 +5,10 @@
 #' [tune_bayes()] for that), and is instead used for fitting a single
 #' model+recipe or model+formula combination across many resamples.
 #'
-#' @param object A workflow, formula, or recipe.
+#' @param object A `parsnip` model specification or a [workflows::workflow()].
```

Comment:
Try using `#' @inheritParams` and fill it out once in `last_fit()`.

Reply:
Addressed in 9ca530512cb153a40454f439161d75f49180e167